.github/ISSUE_TEMPLATE/4_decommission-hub.md

```diff
@@ -40,8 +41,8 @@ Usually, it is because it was a hub that we created for a workshop/conference an
 
 _This phase is only necessary for single hub clusters._
 
-- [ ] Run `terraform plan -destroy` and `terraform apply` to destroy the cluster
+- [ ] Run `terraform plan -destroy` and `terraform apply` from the appropriate workspace, to destroy the cluster
```

Comment:
```suggestion
- [ ] Run `terraform plan -destroy` and `terraform apply` from the [appropriate workspace](https://infrastructure.2i2c.org/en/latest/topic/terraform.html#workspaces), to destroy the cluster
```

Reply:
Good idea @sgibson91! Quick question about the terraform docs, because I just played a bit with it and didn't manage to get `terraform workspace switch to work` the helps shows this:

```new, list, show, select and delete Terraform workspaces.``` 

Is it possible that in the new versions of terraform this option had been removed?

Reply:
I think that's a typo and should be `terraform workspace select`

Reply:
I'll open a PR for that as I think some of the language should be different :)

Reply:
https://github.com/2i2c-org/infrastructure/pull/1020